tests/helpers/test_aiohttp_client.py

```diff
@@ -46,12 +50,10 @@ def camera_client_fixture(hass, hass_client):
         },
     )
     mock_config_entry.add_to_hass(hass)
-    hass.loop.run_until_complete(
-        hass.config_entries.async_setup(mock_config_entry.entry_id)
-    )
-    hass.loop.run_until_complete(hass.async_block_till_done())
+    await hass.config_entries.async_setup(mock_config_entry.entry_id)
+    await hass.async_block_till_done()
```

Comment:
I couldn't work out a reason for this fixture to not be `async`, so I made it `async`.